dired-narrow.el

```diff
@@ -95,6 +95,27 @@ we should act on."
                  (function :tag "Use custom function."))
   :group 'dired-narrow)
 
+(defcustom dired-narrow-exit-when-1-left nil
+  "If there is only one file left while narrowing,
+exit minibuffer and call dired-narrow-exit-action."
```

Comment:
Put functions or variables inside docstrings in the "link" markup, that is start with \` and close with `'`

    call `dired-narrow-exit-action'.

Like so